components/UI/Filters/Filters.module.css

```diff
@@ -0,0 +1,40 @@
+.filters {
+    width: 450px;
+    position: fixed;
+    top: 16px;
+    left: 16px;
+    z-index: 401;
```

Comment:
let's try to reduce the `z-index`
https://github.com/ekaterinburgdev/map/search?q=z-index

apparently, I forgot to make it normal in my PR)

Reply:
400 is minimum. If z-index is less filters disappear because leaflet map's z-index is 400